app/assets/javascripts/discourse/components/backup-uploader.js.es6

```diff
@@ -0,0 +1,55 @@
+import { ajax } from "discourse/lib/ajax";
+import computed from "ember-addons/ember-computed-decorators";
+import UploadMixin from "discourse/mixins/upload";
+
+export default Em.Component.extend(UploadMixin, {
+  tagName: "span",
+
+  @computed("uploading", "uploadProgress")
+  uploadButtonText(uploading) {
```

Comment:
What are you not passing `uploadProgress` as an argument of the `uploadButtonText` function?

```javascript
@computed("uploading", "uploadProgress")
uploadButtonText(uploading, progress) {
  return uploading ? I18n.t("admin.backups.uploading_progress", { progress }) : I18n.t("admin.backups.upload.label");
}
```